_data/pages/home.yml

```diff
@@ -7,8 +7,8 @@ page_last_updated: false
 blocks:
   - type: nav_sticky_banner
```

Comment:
I don't think we really need this from my understanding this should be created from the cms I created it just for the developing stage.